pipeline: output: splunk: style (#1975)
fluent/fluent-bit-docs: pipeline/outputs/splunk.md

}
```

### Sending raw events

> If the option `splunk_send_raw` has been enabled, the user must add all log details in the event field, and only specify fields known to Splunk in the top level event. If there is a mismatch, Splunk returns an HTTP error `400`.

Reviewer: ```suggestion
If the option `splunk_send_raw` has been enabled, the user must add all log details in the event field, and only specify fields known to Splunk in the top level event. If there is a mismatch, Splunk returns an HTTP `400 Bad Request` status code.
```

Consider the following examples:

- `splunk_send_raw` off

  ```json
  {"time": "SOMETIME", "event": {"k1": "foo", "k2": "bar", "index": "applogs"}}
  ```

- `splunk_send_raw` on

  ```json
  {"time": "SOMETIME", "k1": "foo", "k2": "bar", "index": "applogs"}
  ```

For up-to-date information about the valid keys, see [Getting Data In](https://docs.splunk.com/Documentation/Splunk/7.1.10/Data/AboutHEC).

## Splunk metric index

With Splunk version 8.0 and later, you can use the Fluent Bit Splunk output plugin to send data to metric indices. This lets you perform visualizations, metric queries, and analysis with other metrics you might be collecting. This is based off of Splunk 8.0 support of multi metric support using single JSON payload, more details can be found in [Splunk metrics documentation](https://docs.splunk.com/Documentation/Splunk/9.4.2/Metrics/GetMetricsInOther#The_multiple-metric_JSON_format)

> Sending to a Splunk metric index requires the use of `Splunk_send_raw` option being enabled and formatting the message properly. This includes three specific operations

Reviewer: ```suggestion
Sending to a Splunk metric index requires the use of `Splunk_send_raw` option being enabled and formatting the message properly. This includes these specific operations:
```

{% endtab %}
{% endtabs %}

## Send metrics events of Fluent Bit

> In Fluent Bit 2.0 or later, you can also send Fluent Bit metrics the `events` type into Splunk using Splunk HEC.
> This lets you perform visualizations, metric queries, and analysis with directly sent using Fluent Bit metrics.
> This is based off Splunk 8.0 support of multi metric support using a single concatenated JSON payload.

Reviewer: ```suggestion
In Fluent Bit 2.0 or later, you can send Fluent Bit metrics the `events` type into Splunk using Splunk HEC. This lets you perform visualizations, metric queries, and analysis with directly sent using Fluent Bit metrics. This is based off Splunk 8.0 support of multi metric support using a single concatenated JSON payload.
```
